### PR TITLE
Adding turkish foods

### PR DESCRIPTION
**Pull request by bdennisr**
#### Summary
Content "Added Turkish foods with recipes and a cookbook"

#### Purpose of change

#64204 
We added the requested content but more is always welcome!

#### Describe the solution

The content requested was added!

#### Describe alternatives you've considered

N/A

#### Testing

Checked the game to make sure the cookbook was added and that it contained the new food items.

### CODE REVIEW

File: data/json/recipes/food/baking.json [hunk 1]
```diff
@@ -65,5 +65,28 @@
...
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 }, { "id": "OVEN", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
+    "components": [
+      [ [ "walnut_shelled", 4 ] ],
+      [ [ "honey_bottled", 1 ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/baking.json:84]: I'd make this honey or maple syrup here.

File: data/json/recipes/food/meat_dishes.json [hunk 2]
```diff
@@ -23,5 +23,130 @@
...
+    "components": [
+      [ [ "meat_red_raw", 1, "LIST" ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "pepper", 1 ] ],
+      [ [ "onion", 2 ] ],
+      [ [ "tomato", 4 ] ],
+      [ [ "bell_pepper", 2 ] ],
+      [ [ "garlic_clove", 2 ] ],
+      [ [ "seasoning_mild", 2, "LIST" ] ],
+      [ [ "yoghurt", 1 ] ]
```
Comment (Maleclypse) [inline data/json/recipes/food/meat_dishes.json:98]: Not necessarily your problem to fix but if I'm reading the json correctly the only way to make yoghurt is to already have some yoghurt in game currently.   Someone should probably fix that.

**Review by Maleclypse — COMMENTED** [review]
Very cool.

File: data/json/recipes/food/baking.json (hunk 1 above)
Comment (bdennisr) [reply data/json/recipes/food/baking.json:84]: I couldn't find "honey" anywhere as a food item by itself, only as a material but I'm not sure how different that is.
image

File: data/json/items/comestibles/drink_other.json [hunk 3]
```diff
@@ -341,5 +341,50 @@
...
+    "container": "jar_glass_sealed",
+    "sealed": true,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 25,
+    "calories": 280,
+    "description": "A molasses-like syrup obtained after condensing juices of grape must.",
+    "price": 200,
+    "price_postapoc": 200,
+    "material": [ "tomato", "veggy" ],
```
Comment (TheShadowFerret) [inline data/json/items/comestibles/drink_other.json:383]:
```suggestion
    "description": "A molasses-like syrup obtained after condensing juices of grape must.",
    "price": 200,
    "price_postapoc": 200,
    "material": [ "fruit" ],
```
I'm having a big think 🤔

File: data/json/recipes/food/baking.json (hunk 1 above)
Comment (Maleclypse) [reply data/json/recipes/food/baking.json:84]: Sorry I meant add maple syrup as an option as well as the bottled honey you have already there.

Comment (bdennisr) [reply data/json/recipes/food/baking.json]: I see! I added maple_sap as an option!

File: data/json/recipes/food/baking.json [hunk 4]
```diff
@@ -65,5 +65,28 @@
...
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 }, { "id": "OVEN", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
+    "components": [
+      [ [ "walnut_shelled", 4 ] ],
+      [ [ "honey_bottled", 1 ], [ "maple_sap", 1 ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/baking.json:84]:
```suggestion
      [ [ "honey_bottled", 1 ], [ "syrup", 1 ] ],
```

File: data/json/recipes/food/baking.json [hunk 5]
```diff
@@ -65,5 +65,28 @@
...
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 }, { "id": "OVEN", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
+    "components": [
+      [ [ "walnut_shelled", 4 ] ],
+      [ [ "honey_bottled", 1 ], [ "syrup", 1 ] ],
+      [ [ "any_butter", 1, "LIST" ] ],
+      [ [ "sugar", 2 ] ],
+      [ [ "flour_any", 1, "LIST" ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/baking.json:87]:
```suggestion
      [ [ "flour_any", 10, "LIST" ] ],
```

File: data/json/recipes/food/baking.json [hunk 6]
```diff
@@ -65,5 +65,28 @@
...
+  {
+    "type": "recipe",
+    "result": "baklava",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_SNACK",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
```
Comment (Maleclypse) [inline data/json/recipes/food/baking.json:78]:
```suggestion
    "batch_time_factors": [ 80, 1 ],
    "charges": 10,
```

File: data/json/recipes/food/meat_dishes.json [hunk 7]
```diff
@@ -23,5 +23,130 @@
...
+    "components": [
+      [ [ "meat_red_raw", 1, "LIST" ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "pepper", 1 ] ],
+      [ [ "onion", 2 ] ],
+      [ [ "wild_herbs", 1 ] ],
+      [ [ "eggs_bird", 1, "LIST" ] ],
+      [ [ "any_butter", 1, "LIST" ] ],
+      [ [ "garlic_clove", 2 ] ],
+      [ [ "flour_any", 1, "LIST" ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/meat_dishes.json:123]:
```suggestion
      [ [ "flour_any", 20, "LIST" ] ],
```

File: data/json/recipes/food/meat_dishes.json [hunk 8]
```diff
@@ -23,5 +23,130 @@
...
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 40, "LIST" ] ] ],
+    "components": [
+      [ [ "meat_red_raw", 1, "LIST" ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "pepper", 1 ] ],
+      [ [ "onion", 2 ] ],
+      [ [ "wild_herbs", 1 ] ],
+      [ [ "eggs_bird", 1, "LIST" ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/meat_dishes.json:120]:
```suggestion
      [ [ "eggs_bird", 2, "LIST" ] ],
```

File: data/json/recipes/food/meat_dishes.json [hunk 9]
```diff
@@ -23,5 +23,130 @@
...
+  },
+  {
+    "type": "recipe",
+    "result": "manti",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "1 hour",
```
Comment (Maleclypse) [inline data/json/recipes/food/meat_dishes.json:109]:
```suggestion
    "time": "1 hour",
    "charges": 10,
```

**Comment by kevingranade** [comment]
I have a complaint, your PR is making me hungry.

**Comment by kevingranade** [comment]
Also tests are complaining about calories:
`(~[slow] ~[.],starting_items)=> ../tests/comestible_test.cpp:195
(~[slow] ~[.],starting_items)=> ...............................................................................
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> ../tests/comestible_test.cpp:230: FAILED:
(~[slow] ~[.],starting_items)=>   CHECK( lower_bound <= mystats.calories.avg() )
(~[slow] ~[.],starting_items)=> with expansion:
Error: (~[slow] ~[.],starting_items)=>   7500.0f <= 1440.8333333333
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> RecipeID: baklava, default is 10000 Calories,`

File: data/json/items/comestibles/baked.json [hunk 10]
```diff
@@ -37,5 +37,24 @@
...
+  {
+    "type": "COMESTIBLE",
+    "id": "baklava",
+    "name": { "str_sp": "baklava" },
+    "weight": "50 g",
+    "color": "brown",
+    "spoils_in": "7 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 1000,
```
Comment (kevingranade) [inline data/json/items/comestibles/baked.json:50]: https://www.nutritionix.com/i/nutritionix/baklava-1-piece/546c6fd4a589ba77150f2599
Says 306 calories for a 76g serving, that would put this at about 200 calories, unless im missing something about e.g. stack size.
```suggestion
    "calories": 200,
```

**Comment by kevingranade** [comment]
After that major correction it's very close to being happy now:
```
Error: (~[slow] ~[.],starting_items)=>   1500.0f <= 1440.8333333333
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> 
(~[slow] ~[.],starting_items)=> RecipeID: baklava, default is 2000 Calories,
(~[slow] ~[.],starting_items)=> Current recipe range: 1361-1535, Average 1440.8, Stddev 64.7
(~[slow] ~[.],starting_items)=> Average recipe Calories must fall within this range, derived from default Calories: 1500-2500
```
Basically it's looking at the different ingredients you can use to make this recipe and adding up their calories,  and the result should be pretty close to what we've said that many (10) servings of baklava should have.
I think if we bump the number of servings the recipe makes to 12-16, I recommend the middle of the range so 14, it will be totally happy. 
It's possible a better fix would be adjusting the ratios of ingredients, but that's a bit more complex so I'm not getting into that.

File: data/json/recipes/food/baking.json [hunk 11]
```diff
@@ -65,5 +65,29 @@
...
+    "charges": 10,
```
Comment (kevingranade) [inline data/json/recipes/food/baking.json:79]: Unless I'm missing something,  this should make the recipe sanity check pass.
```suggestion
    "charges": 14,
```

File: data/json/recipes/food/baking.json [hunk 12]
```diff
@@ -65,5 +65,29 @@
...
+    "type": "recipe",
+    "result": "baklava",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_SNACK",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
+    "charges": 14,
```
Comment (kevingranade) [inline data/json/recipes/food/baking.json:79]: I'm a dumb, I adjusted the wrong way. 
```suggestion
    "charges": 7,
```

**Pull request merged**